examples/SimpleConsumer/Program.cs

```diff
@@ -38,13 +38,19 @@ public static void Main(string[] args)
                 { "bootstrap.servers", brokerList }
             };
 
-            using (var consumer = new Consumer<Null, string>(config, null, new StringDeserializer(Encoding.UTF8)))
+            using (var consumer = new Consumer<Ignore, string>(config, null, new StringDeserializer(Encoding.UTF8)))
             {
                 consumer.Assign(new List<TopicPartitionOffset> { new TopicPartitionOffset(topics.First(), 0, 0) });
+                
```

Comment:
It'd be nice with a comment that tells the user what the difference is between these two. Global errors (disconnects, etc) vs consume stream errors (failure to commit, etc).

Reply:
good idea.